### PR TITLE
Update Spring Boot to 2.2.6.RELEASE

### PR DESCRIPTION
**Pull request by jacobdotcosta**

### CODE REVIEW

**Comment by jacobdotcosta** [comment]
> This doesn't seem to be using the latest version of the examples… For example, `health-check` should be at `2.2.6-5`.

You are right. Looks like the automation client didn't all it had to do.

**Comment by metacosm** [comment]
Another aspect that I'm not sure about is whether the 2.2 version should replace the 2.1 one or if we should keep both. What do you think, @zekaguilar?

File: metadata.json [hunk 1]
```diff
@@ -187,11 +187,11 @@
       "versions": [
         {
           "id": "current-community",
-          "name": "2.1.13.RELEASE (Community)"
+          "name": "2.2.6.RELEASE (Community)"
         },
         {
           "id": "current-redhat",
-          "name": "2.1.13.RELEASE (RHOAR)"
+          "name": "2.2.6.RELEASE (RHOAR)"
```
Comment (metacosm) [inline metadata.json:194]: I thought that we were supposed to change the supported release name? @cmoulliard @zekaguilar

Comment (geoand) [reply metadata.json:194]: Good point

File: catalog.json [hunk 2]
```diff
@@ -1026,7 +1026,7 @@
     "name": "Istio - Spring Boot - Distributed Tracing",
     "description": "Example to demonstrate how Istio's service mesh and Distributed Tracing - Jaeger interact.",
     "repo": "https://github.com/snowdrop/istio-distributed-tracing-example",
-    "ref": "2.1.6-9",
+    "ref": "2.2.6-5",
```
Comment (metacosm) [inline catalog.json:1029]: Did we miss updating this example for 2.1.13?

**Comment by metacosm** [comment]
Example versions look OK to me, otherwise.

**Comment by zekaguilar** [comment]
Hi @metacosm , let's wait until Monday meeting to see the options and we will take a decision. If the change involves QE effort I will leave as it is now and look for to update in SP2

**Comment by quintesse** [comment]
[test]

**Comment by metacosm** [comment]
> Hi @metacosm , let's wait until Monday meeting to see the options and we will take a decision. If the change involves QE effort I will leave as it is now and look for to update in SP2

This is a purely PM decision: there is no QE involved, just deciding what we want to offer to our users. Everything is QE’d already once it gets to this point.

**Comment by zekaguilar** [comment]
Deferred to SP1

**Pull request closed without merging**